bin/e2e-setup

```diff
@@ -1,3 +1,9 @@
+if [ -z "$E2E" ]
```

Comment:
**nitpick (non-blocking):** FWIW, the most common style for this shell scripts is

```sh
if [ -z "$E2E" ]; then
  ...
fi
```

People who put `then` on the next line generally indent the branch even further.

Reply:
Done.